ui/webui/src/components/storage/StorageConfiguration.jsx

```diff
@@ -111,7 +111,7 @@ const scenarios = [{
     initializationMode: 0,
     buttonLabel: _("Install"),
     buttonVariant: "primary",
-    screenWarning: "",
+    screenWarning: "To prevent loss, make sure to backup your data.",
```

Comment:
Please translate these.

Reply:
Oh no, I really went backwards since I stopped coding and forgetting to translate things :)

Reply:
Fixed